pkg/haproxy/config.go

```diff
@@ -85,15 +85,29 @@ frontend stats
   stats refresh 500ms
   stats hide-version
   stats show-legends
-{{ $port := .Port | printf "%d" }}
+{{range .ExtraServices}}
+frontend svc_{{.Name}}
+  mode tcp
+  bind *:{{.Port | printf "%d"}} name lb
+  option tcplog
+  default_backend svc_{{.Name}}
+{{end}}{{$port := .Port | printf "%d"}}
 backend kube_api_backend
   mode tcp
   balance first
   option httpchk GET /readyz
   default-server inter 10s downinter 10s rise 5 fall 3 slowstart 120s maxconn 1000 maxqueue 256 weight 100{{range .Addresses}}
   server {{ .NodeName }} {{ .IP }}:{{ $port }} check check-ssl verify none{{end}}
   http-check expect status 200
-
+{{ range .ExtraServices }}{{ $backendPort := .Backend.Port | printf "%d" }}{{ $backendHealthCheckConfig := .Backend.HealthCheckConfig }}
+backend svc_{{.Name}}
+  mode tcp
+  balance roundrobin{{if $backendHealthCheckConfig.Option}}
```

Comment:
Perhaps move the if conditional into next line? You can always use `-` to suppress blank lines.

Reply:
Thanks for the tip. Didnt know about "-".